entry/streamer-entry

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+
+SELF=$(basename $0)
+
+function err() {
+  echo "ERR $SELF $$ ${FUNCNAME[1]}: $@" >&2
+}
+
+function dbg() {
+  echo "DBG $SELF $$ ${FUNCNAME[1]}: $@" >&2
+}
+
+function help() {
+    cat <<EOF
+Usage: $SELF [-i <id>] [-u <uid>]  [-h] <url>
+  Streamer provide streaming service between AIC and WebRTC client
+
+  -i <id>: instance ID, default: $id
+  -u <uid>: user id, default: 0
+  -l:      enable measuring latency
+  -h:      print the usage message
+  <ip>:    ip to signaling server in format
+
+EOF
+}
+
+function main() {
+  local uid=0
+  local w=${K8S_ENV_DISPLAY_RESOLUTION_X:-1280}
+  local h=${K8S_ENV_DISPLAY_RESOLUTION_Y:-720}
+  local y=${ICR_ON:-n}
+  local user_count=${K8S_ENV_USER_COUNT_PER_ANDROID:-1}
+  local dev=${DEVICE:-/dev/dri/renderD128}
+  local cqp=${K8S_ENV_FORCE_CQP:-false}
+  local wdir="/opt/workdir"
+  local k8s=${K8S_ENV:-"false"}
+  local k8s_stateless=${K8S_ENV_STATELESS:-"false"}
+  local dev=${DEVICE:-/dev/dri/renderD128}
+  local id="$(id)"
+  local measure_latency="false"
+  local privileged_enable=${ENABLE_ENV_PRIVILEGED:-"false"}
+  local codec=${CODEC:-"avc"}
+  local icr_start_immediately=${ENV_ICR_START_IMMEDIATELY:-"false"}
+
+  while getopts ":i:u:lh" opt; do
+    case $opt in
+    i) id=$OPTARG ;;
+    u) uid=$OPTARG;;
+    l) measure_latency="true";;
+    h) help && exit 1 ;;
+    *) echo "no such option: $opt" && exit 1 ;;
+    esac
+  done
+  shift $((OPTIND - 1))
+
+  if [[ -n "$1" ]]; then
+    ip="$1"
+  else
+    err "signaling server ip is required"
+    help
+    exit 1
```

Comment:
Ditto.  You could simplify code by moving error message printing and exiting to (renamed) `help()` function.